test/test_torch.py

```diff
@@ -1645,6 +1645,39 @@ def test_to_with_tensor(self):
                 self.assertEqual(a.device, b.to(a).device)
                 self.assertEqual(b.device, a.to(b).device)
 
+    def test_to_with_requires_grad(self):
+        # only requires_grad
+        a = torch.tensor(5.)
+        self.assertEqual(a, a.to(requires_grad=True))
```

Comment:
Is this correct? I thought that the returned tensor should be a different object (out-of-place).

Reply:
It is a different object, but it is equal (we don't check autograd flags when testing equality)